### ISSUE TITLE
R1 SN003 new back cover doen't properly close

### ISSUE THREAD

**Issue by lnobile** [robotology/icub-tech-support#1131]
### Robot name 🤖

R1 SN003

### Request/Failure description

The new R1 SN003 new back cover doen't properly close since it has an interference with the nvidia xavier board mounted on the robot.

### Detailed context

@fbiggi already cutted out a part of the cover to make it fit so this is no more an issue but just a remainder to update the drawings in the future.
The part cutted (see photo) was in contact with the fan housing of the Nvidia xavier board.
![IMG_8522](https://user-images.githubusercontent.com/56638910/117001757-049ec600-ace3-11eb-909e-8ff021978a1e.jpg)

**Comment by fbiggi** [comment]
the cover actually strained on the rear fan
I preferred to cut and shape
It seems that closing is much easier

cc @fiorisi @maggia80 @Fabrizio69

**Comment by fiorisi** [comment]
@lnobile having a look at the CAD model it seems that there shouldn't be any interference. I think that we should understand which is the difference between the CAD model and the real prototype.
![image](https://user-images.githubusercontent.com/8970230/117004839-d3c09000-ace6-11eb-8f9b-46718b7730bf.png)
![image](https://user-images.githubusercontent.com/8970230/117004875-e20eac00-ace6-11eb-9688-3eff57473bea.png)
![image](https://user-images.githubusercontent.com/8970230/117005051-17b39500-ace7-11eb-830c-f75503050df0.png)

**Comment by lnobile** [comment]
It seems to me that the upper ribbon of the cover is at the same height of the nvidia xavier's fan.
I don't know if the ribbon is lower or the fan higher, but this seems the problem.
![IMG_8525](https://user-images.githubusercontent.com/56638910/117007157-97dafa00-ace9-11eb-88e8-c046ae626829.jpg)

**Comment by lnobile** [comment]
The board in CAD seems below the white plastic plate under the shoulders motors, while on the real robot the upper part of the board is at the same height of the white plate

**Comment by lnobile** [comment]
Ok I got it, the board is mounted upside down (look at the connectors)

**Comment by maggia80** [comment]
yes! Indeed the board is mounted upside down. If there aren't specific reasons we could put it in the "right" position.

**Comment by lnobile** [comment]
It doesn't seem to me, connector cables have all sufficient length.
Maybe @randaz81 or @emilianob80 could know any possible reason

**Comment by randaz81** [comment]
> Maybe @randaz81 or @emilianob80 could know any possible reason
No idea. Are you sure that every connector (usb/network/hdmi) have enough space for insertion? Eventually we might need to buy cables with angled connectors.

**Comment by fbiggi** [comment]
la casa di R1é in lab
@MrAndrea @marcellosavoldi non posso fare le prove di imballaggio con la cover nuova
Quando potete darmelo rimontato?

**Comment by MrAndrea** [comment]
@fbiggi oggi dovremmo riuscire a richiudere la testa di R1. @violadelbono sta preparando i cavetti per gli speaker e per l'amplificatore audio, e non appena pronti procederemo al cablaggio e al test e a rimontare le cover.

cc @marcellosavoldi @maggia80

**Comment by MrAndrea** [comment]
Nel pomeriggio con @violadelbono ho collegato l'amplificatore ma purtroppo gli speaker gracchiano molto e di fatto l'audio si sente poco e male. Provando a cambiare un po' i collegamenti per trovare la causa del problema il risultato è stato sempre lo stesso, e sembra indipendente dal volume che regola dal pc con l'alsamixer.
Infine anche con @maggia80 ho riprovato i test ma visti gli stessi risultati abbiamo deciso di ripristinare il vecchio collegamento e richiudere la testa.

@fbiggi il robot è rimontato e puoi provare la cassa.

**Comment by fbiggi** [comment]
ATTENZIONE !!!
Con @lnobile abbiamo provato a mettere il robot R1 con la back cover nuova per XAVIER
IL robot tocca e scarica il peso sulla cover nuova ...e la pedana non si riesce a togliere.
Senza cover la XAVIER tocca e potrebbe danneggiarla.
Abbiamo deciso dopo il lavoro di @lnobile  con il team di smontare la XAVIER 
Datemi info e conferme

@maggia80 @randaz81 @Fabrizio69 @emilianob80 @violadelbono @fiorisi
